skbio/sequence/_rna.py

```diff
@@ -136,6 +136,72 @@ def _motifs(self):
         return _motifs
 
     @stable(as_of="0.4.0")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        Uracil (U) is replaced with thymine (T) in the reverse transcribed
+        sequence.
+
+        Returns
+        -------
+        DNA
+            Reverse transcribed sequence.
```

Comment:
I think the parenthetical on the return description is really all we need. It's just to confirm the common sense assumption about the return.